docs/versioned_docs/version-zenith/quickstart/532928-up.mdx

```diff
@@ -0,0 +1,30 @@
+---
+slug: /user/quickstart/532928/up
+hide_table_of_contents: true
+title: "Expose services returned by a function on the host"
+---
+import PartialExperimentalDocs from '../partials/_experimental.mdx';
+
+# Quickstart
+
+<PartialExperimentalDocs />
+
+## Expose services returned by a function on the host
+
+The `dagger up` command allows a service returned by a function to be executed and have any exposed ports forwarded to the host machine. This has many potential use cases, such as manually testing web servers or databases directly from the host browser or host system.
```

Comment:
Know you mentioned you're still updating this @vikram-dagger, so may already be on your radar, but just in case not the commands changed to be e.g. `dagger call package-service up --port 8080:80`

Reply:
Thanks! This section will be dropped so not making this change here but noted for other pages.